docs/metrics/index.md

```diff
@@ -0,0 +1,132 @@
+how it will answer them, what success looks like (that is, how do we know when
+we're done?), and to help narrow an experiments scope so it focuses on what
+will truly answer the question.  Often we'll come into an experiment with ideas
+on how to measure something and after writing this document realize that there
+are better options.
+
+The document is expected to be at the `docs/metrics.md` path from the root of
+the repository.  Ideally it includes actual sketches of dashboard visuals the
+experiment leads would like to see.  It should include any special
+circumstances or requirements in the experiment, as well as details on how the
+metrics will be collected (keys, schemas, names of elements, etc.)
+
+Several examples exist
+([1](https://github.com/meandavejustice/min-vid/blob/master/docs/metrics.md),
+[2](https://github.com/mozilla/universal-search/blob/master/docs/metrics.md),
+[3](https://github.com/internetarchive/FirefoxNoMore404s/blob/master/docs/metrics.md))
+but it's important to understand that this is an evolving process and these
+examples may not represent the current needs (for example, none of the above
+leverage Google Analytics).
+
+The Metrics plan should be looked at by the Mozilla Measurement & Analytics
+Teams to make sure avoid implementation troubles.
+
+# Implementing Metrics in an Experiment
+
+The Metrics Plan you've written should serve as a blueprint for implementation.
+
+## Telemetry
+To use Telemetry you'll need to make sure the Mozilla Measurement Team is aware
+a new experiment is starting up.  That involves filing a bug to get Test Pilot
+Experiment data into re:dash, and then creating a re:dash dashboard.  Feel free
+to [clone these two bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=1297199%2C+1297200).
+
+The Measurement Team will be looking at the Metrics Plan you wrote to
+understand what is needed so it's important it includes the schema you'll be
+logging your data to.
+
+The actual function to ping Telemetry are provided by the Test Pilot add-on.
+Same code is provided below:
+
+----------------------------------------------------------------------------
+
+TODO: Please help me write good sample code here.  Ideally in a try/catch for if
+the test pilot add-on is not installed at all.
+```javascript
+   // Sample code here
+```
+
+Do we have a recommended way to ping already?  It looks like the experiments
+are doing it in substantially different ways. eg:
+
+* https://github.com/meandavejustice/min-vid/blob/master/lib/send-metrics-data.js
+* https://github.com/internetarchive/FirefoxNoMore404s/blob/master/src/scripts/background.js#L108
+
+TODO
+
+----------------------------------------------------------------------------
+
+
+## Google Analytics
+To use Google Analytics we recommend you use your own Google Analytics account
+to record data.  This makes the transition easy when your experiment graduates
+from the Test Pilot program.
```

Comment:
Because this is the case, should we mention permissions? I'd think we want to make sure that the Test Pilot team has at least read access to each of these accounts.